### ISSUE TITLE
Music plays on its own on other computers

### ISSUE THREAD

**Issue by Sunlis** [animal-crossing-music-extension/ac-music-extension#132]
I have two computers signed into the same Google account in Chrome. If one of them is playing music at the top of the hour, the other computer will also start playing music.

**Comment by PikaDude** [comment]
I don't believe this is something we can really control unfortunately. Chrome has in-built functionality to synchronise your extension settings and storage across devices, but I don't believe extensions themselves can tell Chrome what and what not to sync. I don't believe this is something we can fix, however you may find luck in disabling extensions from syncing in Chromes sync settings.
